staging/src/k8s.io/apiserver/pkg/registry/rest/rest.go

```diff
@@ -99,6 +98,8 @@ type Lister interface {
 	NewList() runtime.Object
 	// List selects resources in the storage which match to the selector. 'options' can be nil.
 	List(ctx context.Context, options *metainternalversion.ListOptions) (runtime.Object, error)
+
+	TableConvertor
```

Comment:
this ensures all listers provide this interface

Reply:
Ok

Reply:
Probably deserves a comment